.github/workflows/rust.yml

```diff
@@ -198,26 +219,26 @@ jobs:
         uses: ./.github/actions/setup-builder
         with:
           rust-version: stable
+      - name: Check datafusion-functions (default features)
+        run: cargo check --profile ci -p datafusion-functions
```

Comment:
I don't think there is any reason to check all targets (tests as well) so I kept just the tests for the library